crates/astria-sequencer/src/app/mod.rs

```diff
@@ -1619,7 +1591,7 @@ impl App {
         self.upgrades_handler
             .upgrades()
             .aspen()
-            .map_or(true, |aspen| {
+            .map_or(false, |aspen| {
```

Comment:
update comment on this function:
> the assumption is that this network uses `DataItem`s from genesis onwards.
should be that this network does not use `DataItem`s?

Reply:
Fixed in [6719cb2](https://github.com/astriaorg/astria/pull/2093/commits/6719cb290cc36b58a891a3afbaa27ba0c7415696).